Tools/BuildAndPackage.ps1

```diff
@@ -81,3 +108,11 @@ Compress-Archive -Path @($powerShellDir, $gcpsDir) $archivePath
 Write-Host -ForegroundColor Cyan "*** Complete ***"
 Write-Host "The latest build is at:"
 Write-Host $archivePath
+
+Write-Host "Generating Website"
+$generateWebsiteFile = (Get-ChildItem "$PSScriptRoot\GenerateWebsiteData.ps1").FullName
+# We do this on another process because the GenerateWebSiteData script will load the module, so in case we want to
```

Comment:
One problem with spawning another process is that if the GenerateWebsiteData script fails, we don't get any feedback about it. It might be worth-while to put in a sanity check to confirm that `cmdletsFull.json` was created/modified in the last 5 minutes. But that's a lower-priority.